felix/dataplane/linux/dscp_mgr.go

```diff
@@ -132,12 +146,20 @@ func (m *dscpManager) handleWEPUpdates(wepID *proto.WorkloadEndpointID, msg *pro
 func normaliseSourceAddr(addrs []string) string {
 	var trimmedSources []string
 	for _, addr := range addrs {
-		parts := strings.Split(addr, "/")
-		trimmedSources = append(trimmedSources, parts[0])
+		trimmedSources = append(trimmedSources, removeSubnetMask(addr))
 	}
 	return strings.Join(trimmedSources, ",")
 }
 
+func removeSubnetMask(addr string) string {
+	// addr is in format of a.b.c.d/x
+	parts := strings.Split(addr, "/")
+	if len(parts) == 0 {
+		logrus.Panicf("Malformed address %s", addr)
```

Comment:
Not sure we want to have panics was down in felix - probably best to simply log an error and ignore the address.

I know it should be validated, but best not to accidentally take down the cluster if we miss something!

Reply:
Fixed it.